crates/oxc_transformer/src/es2018/object_rest_spread.rs

```diff
@@ -992,7 +994,7 @@ impl<'a> ObjectRestSpread<'a, '_> {
                 // `let { [1], ... rest }`
                 if expr.is_literal() {
                     let span = expr.span();
-                    let s = expr.to_js_string().unwrap();
+                    let s = expr.to_js_string(&WithoutGlobalReferenceInformation {}).unwrap();
```

Comment:
For the places that linter and transformer was using, I passed `WithoutGlobalReferenceInformation` for now which treats all identifiers as unknown.